src/components/global-styles/GlobalStyles.tsx

```diff
@@ -5,33 +5,30 @@ export const typography = css`
   /* only take latin chars to reduce bundle size */
   @font-face {
     font-family: "Halyard";
+    src: url("/fonts/HalyardDisplay-Regular.woff2") format("woff2"),
+      url("/fonts/HalyardDisplay-Regular.woff") format("woff");
+    font-weight: normal;
     font-style: normal;
-    font-weight: 400;
-    font-display: fallback;
-    src: url("/fonts/HalyardDisplay-Regular.woff2") format("woff2");
-    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA,
```

Comment:
I think this should be kept in, what it does is it fetches only latin characters for this font, reducing bundle size

Reply:
Sounds good. I just pilfered what we had in Umaverse, but I can keep the unicode range.